career/em-ladder.md

```diff
@@ -54,6 +56,7 @@ client projects.
 - [ ] Empowers direct reports to own their work.
 - [ ] Ensures direct reports are aware of growth and learning opportunities and
   helps them leverage these.
+- [ ] Empowers direct reports to speak up about their pain points.
```

Comment:
Isn't this one redundant?

Reply:
@hcarreras 

With the previous one added?

Reply:
Yeap. I was wondering what's the difference between those 2 points? Or in which scenario a person would only mark one of them (or be working on only of the 2).

Reply:
This first scenario is where the EM, is trying to identify their direct reports pain points.
"X is clearly stressed, The EM can see by X's harvest they are jumping projects 7 times a day" 

Where the other one would be encourage their direct reports to speak up about their pain points.
I know it's quite common for junior devs, to be hesitant to speak up when they have pain points, which is why i think there is quite the difference :)

Please let me hear your thoughts.